docs/users-guide/models.md

```diff
@@ -0,0 +1,107 @@
+# Models
+
+Models are a fundamental building block in Metabase. Models curate data from another table or tables from the same database to anticipate the kinds of questions people will ask of the data. You can think of them as derived tables, or a special kind of saved question meant to be used as the starting point for new questions.
```

Comment:
Feels like we could better emphasize that they can be based on a SQL/notebook query, and therefore can have custom/calculated/synthetic columns, which are a big plus.